src/components/Dashboard/FilingHistoryList/filings/ContinuationIn.vue

```diff
@@ -5,32 +5,57 @@
     :index="index"
   >
     <template #subtitle>
-      <FiledAndPendingPaid
+      <SubtitleFiledAndPendingPaid
         v-if="isFutureEffectivePending"
         class="item-header-subtitle"
         :filing="filing"
         :index="index"
       />
 
-      <FutureEffectivePaid
+      <SubtitleFutureEffectivePaid
         v-else-if="isFutureEffective"
         class="item-header-subtitle"
         :filing="filing"
         :index="index"
       />
+
+      <SubtitleRejected
+        v-else-if="isStatusRejected"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
     </template>
 
     <template #body>
-      <FutureEffectivePending
+      <BodyFutureEffectivePending
         v-if="isFutureEffectivePending"
         :filing="filing"
       />
 
-      <FutureEffective
+      <BodyFutureEffective
         v-else-if="isFutureEffective"
         :filing="filing"
       />
 
+      <!-- rejected bootstrap filing -->
+      <div
+        v-else-if="isStatusRejected"
+        class="rejected-continuation-in-details"
+      >
+        <p class="mt-0">
+          This {{ filing.displayName }} is rejected for the following reasons:
+        </p>
+        <p>
+          {{ filing.latestReviewComment || '[staff rejection message]' }}
```

Comment:
do we not need to show the lastReviewComment when change requested?

Reply:
Yes, we do. It's in the Todo List -- see line 613.